### DISCUSSION TITLE
[Feature request] Reduce volume

### DISCUSSION THREAD

**Discussion by cvzi** [PaulWoitaschek/Voice#1301] [Ideas]
I would love to see an option to decrease the volume of an audio book, i.e. the opposite of the volume boost/gain slider. 
Some of my audiobooks (or maybe my phone) are too loud even though I have set the volume to minimum.

**Comment by ghost** [comment]
@cvzi this sounds like more of an issue with your phone because I've never heard of any phone playing audio files "too loudly" at the lowest possible volume settings. No other player does what  you want so I don't see why Voice should do it.

**Comment by cvzi** [comment]
My current phone (Moto G6) on minimal volume is about as loud as my last phone (Samsung S3) on the center position, so the phone is a big factor.
It seems to be a common problem with Android phones, at least there are a lot of google results and many hackish workaround apps/solutions. I guess the problem is that standard Android has only 14 levels, which are too coarse on louder phones.

**Comment by ghost** [comment]
@cvzi That seems to be a problem with your specific phone. This thread found a solution for your issue. 

https://forums.lenovo.com/t5/Moto-G6-Moto-G6-Plus-Moto-G6/Re-Minimum-volume-is-way-too-loud/td-p/4172984

**Comment by cvzi** [comment]
Thanks, but that doesn't work on my phone. I don't hear a difference at all.
Edit: It does work but only for the built in speaker not if headphones are connected.

**Comment by ghost** [comment]
@cvzi did you try looking for a headphone specific setting?

**Comment by cvzi** [comment]
I don't think there is one. 
Do you know how such a feature could be implemented in Voice? With a hint or two I could probably hack it together just for myself. I guess it could be done with the same methods as the fade out at the end of the sleep timer?

**Comment by PaulWoitaschek** [comment]
#833 Will add a circular progress bar which will be used for the volume boost.
This ticket will:
* Rename `Volume Boost` to sth. like `Volume`
* Change the current `db` boost to a percentage boost

The loudnessgain could be removed:
https://github.com/PaulWoitaschek/Voice/blob/2e9e338853c53159c1208e3575035e2d6d07bbf2/app/src/main/java/de/ph1b/audiobook/playback/MediaPlayer.kt#L185

And merged with the volume:
https://github.com/PaulWoitaschek/Voice/blob/65796a9125b08bc37ea6a41be14b66376ac25080/app/src/main/java/de/ph1b/audiobook/playback/MediaPlayer.kt#L150

* If volume change is set to `<=1.0` the LoudnessGain will not be used but just the volume.
* If volume is set to > 1.0, the volume is set to `1.0` and the boost will be applied using the LoudnessGain.
